scene/gui/menu_bar.cpp

```diff
@@ -758,7 +757,10 @@ Size2 MenuBar::get_minimum_size() const {
 		}
 		Size2 sz = menu_cache[i].text_buf->get_size() + style->get_minimum_size();
 		size.y = MAX(size.y, sz.y);
-		size.x += sz.x + hsep;
+		size.x += sz.x;
+	}
+	if (menu_cache.size() > 0) {
```

Comment:
```suggestion
	if (menu_cache.size() > 1) {
```

Can probably be like that, not that it matters much.